app/_hub/kong-inc/upstream-tls/index.md

```diff
@@ -2,22 +2,26 @@
 
 name: Upstream TLS
 publisher: Kong Inc.
-version: 1.3-x
+version: 0.36-x
 
 desc: Add TLS to your Services
 description: |
   Enable TLS on upstream traffic by providing Kong with a list of trusted
   certificates.
 
+  <div class="alert alert-warning">
+    <p><strong>This plugin has been deprecated.</strong></p>
+    <br>
+    <p><strong>Starting with <a href="https://docs.konghq.com/enterprise/changelog/#changes-2">Kong 1.3.0.0</a>:</strong></p>
+    <p>To configure Upstream TLS, use the NGINX directives `proxy_ssl_trusted_certificate`, `proxy_ssl_verify`, and `proxy_ssl_verify_depth` instead of the Upstream TLS plugin. This plugin is <strong>only functional for Enterprise versions 0.35 and 0.36</strong>.</p>
```

Comment:
```suggestion
    <p>To configure Upstream TLS, use the NGINX directives <code>proxy_ssl_trusted_certificate</code>, <code>proxy_ssl_verify</code>, and <code>proxy_ssl_verify_depth</code> instead of the Upstream TLS plugin. This plugin is <strong>only functional for Enterprise versions 0.35 and 0.36</strong>.</p>
```